### DISCUSSION TITLE
dashboard is loaded without css styles

### DISCUSSION THREAD

**Discussion by AndryAsh** [openwisp/docker-openwisp#346] [General]
When I go to https://dashboard.mydomain.local I get a page with no css styles.
![dashboard_login](https://github.com/user-attachments/assets/ff366d53-1876-4888-9e3f-4951079baaff)
How can I fix it?

**Comment by nemesifier** [comment]
Please reply to this question in the other thread you just opened: https://github.com/orgs/openwisp/discussions/907#discussioncomment-10349995 .

**Reply by AndryAsh** [reply]
Don't you have release numbers or container release numbers?
How do I determine the version of a container?
I downloaded the latest versions and the OSS styles don't work.
Answer me please.
